app/assets/stylesheets/organisms/_document-preview.scss

```diff
@@ -1,36 +1,28 @@
-.document-previewer {
-  display: inline-block;
-  margin-bottom: 1em;
+.doc-preview-container {
```

Comment:
tough call. It seems like in some places in the app we'd have to call it "document" since "doc" can have many meanings and might confuse people. Would be happy to have less typing, though. We can punt for now :)